src/plugins/embeddable/public/lib/panel/embeddable_panel.tsx

```diff
@@ -234,6 +236,8 @@ export class EmbeddablePanel extends React.Component<Props, State> {
         paddingSize="none"
         role="figure"
         aria-labelledby={headerId}
+        hasBorder={this.props.hasBorder}
+        hasShadow={this.props.hasShadow}
```

Comment:
Why did we need to add border and shadow props to the embeddable panel for the anywhere feature?

Reply:
It was needed so we can have them removed/invisible in the view events flyout embeddables. Otherwise there was unnecessary lines and borders in the flyout that we didn't want and weren't part of the UX mocks.

Reply:
got it - do we actually need to use panels within the flyout? I seem to remember that there are other options for embeddables without using the panel wrapper, which sounds closer to what we want.

Reply:
Ah right - I learned about this yesterday actually, let me look into it: https://github.com/opensearch-project/alerting-dashboards-plugin/pull/553

Reply:
Using the renderer seems to break the charts from being visible in the flyout. My assumption is the way that we have the panels wrapped within flex items doesn't work the same for the renderer and causes sizing or positioning issues.

For now, prefer to leave as-is with the panel wrapper. It ensures consistent look/feel to the vis panels as seen in the dashboard, and the modification to use the renderer may require refactoring and effort to get the sizing & positioning to work as expected.

Reply:
That seems fine. It's an aspect of embeddables I'd love for us to get right so it's easy for plugins to embed in either an empaneled or clean state. I'll open a tracking issue.